theano/gpuarray/tests/test_dnn.py

```diff
@@ -656,55 +666,62 @@ def test_conv3d_none(self, border_mode, conv_mode):
                         border_mode,
                         conv_mode,
                         [(1, 1, 1), (2, 2, 2)],
+                        [(1, 1, 1)],
```

Comment:
Ah, I had tried increasing it before by a small amount as you suggested above and that didn't work, but I went and computed a size that should work (10,2,15,16,17) and it works fine.